lstchain/image/tests/test_modifier.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import numpy as np
+from scipy.sparse import csr_matrix
+
+
+@pytest.mark.parametrize('fraction', [0.2, 0.5])
+def test_psf_smearer(fraction):
+    from lstchain.image.modifier import random_psf_smearer, set_numba_seed
+
+    set_numba_seed(0)
+
+    # simple toy example with a single 7 pixel hexgroup
+    image = np.zeros(7)
+
+    # really large number so we can make tight checks on the distributed photon counts
+    # 0 is the central, then clockwise
+    image[0] = 1e4
+
+    neighbor_matrix = csr_matrix(np.array([
+       #0  1  2  3  4  5  6
+       [0, 1, 1, 1, 1, 1, 1],  # 0
+       [1, 0, 1, 0, 0, 0, 1],  # 1
+       [1, 1, 0, 1, 0, 0, 0],  # 2
+       [1, 0, 1, 0, 1, 0, 0],  # 3
+       [1, 0, 0, 1, 0, 1, 0],  # 4
+       [1, 0, 0, 0, 1, 0, 1],  # 5
+       [1, 1, 0, 0, 0, 1, 0],  # 6
+    ]))
+
+
+    smeared = random_psf_smearer(image, fraction, neighbor_matrix.indices, neighbor_matrix.indptr)
+    # no charge lost in this case
+    assert image.sum() == smeared.sum()
+
+    # test charge is distributed into neighbors
+    assert np.isclose(smeared[0], (1 - fraction) * image[0], rtol=0.01)
+    assert np.allclose(smeared[1:], image[0] * fraction / 6, rtol=0.1)
```

Comment:
this one here is working because you are testing it with an array with 7 elements, it kept me confused because I was thinking on `smeared[1:]` as an array of 1854 elements... maybe some test with a real geometry?

Reply:
Why? This is a well-working unit test that allows as to test much more  strictly than if we would use a much more complicated geometry.

Reply:
@maxnoe : where is the test called, and with what "fraction"?  I can't find it. With the random version the tolerance seems a bit small to me.

Reply:
@moralejo it is at the top of the test, the test is parameterized to run with fractions 0.2 and 0.5

Reply:
The tolerance works fine for the high value of charge I injected here.